.github/workflows/ci-extended.yml

```diff
@@ -66,12 +66,31 @@ jobs:
           cd build
           ctest -L regression -L ${{ matrix.parallel }} -LE perf-reg --timeout 3600
 
+      # Test Ascent integration (only most complex setup with MPI and on device)
+      - name: Ascent tests
+        if: ${{ matrix.parallel == 'mpi' && matrix.device == 'cuda' }}
+        run: |
+          cmake -B build-ascent \
+            -DCMAKE_BUILD_TYPE=Release \
+            -DMACHINE_VARIANT=${{ matrix.device }}-${{ matrix.parallel }} \
+            -DPARTHENON_ENABLE_ASCENT=ON \
+            -DAscent_DIR=/usr/local/ascent-develop/lib/cmake/ascent
+          cmake --build build-ascent
+          cd example/advection/
+          mpirun -np 2 ../../build-ascent/example/advection/advection-example \
+            -i parthinput.advection \
+            parthenon/output4/dt=0.05 \
+            parthenon/time/tlim=0.1
+          # check if file exists
+          test -f ascent_render_57.png
```

Comment:
So we just check that Ascent works? Probably not wise to do a bitwise check that it matches?

Can Ascent produce other data products that can be compared bitwise, would that be a more robust test? Don't hold up the PR to implement that check though.

Reply:
We could add something like idiff (https://openimageio.readthedocs.io/en/latest/idiff.html) or perceptualdiff (https://pdiff.sourceforge.net/) to compare against a reference image output?

Reply:
Yes, I'd be happy to have sth like this (also for other tests), but I thought in the first round, this is beyond this PR (as it might require additional [unrelated] changes in the test infrastructure as well)

Reply:
Ok, fine with me.